src/ui/navigation.js

```diff
@@ -71,6 +71,7 @@ const NavElement = ({ target, children, icon, ...rest }) => {
 const Navigation = props => {
   const [isOpened, updateIsOpened] = useState(false);
   const toggleMenu = () => updateIsOpened(!isOpened);
+  const closeMenu = () => updateIsOpened(false);
```

Comment:
Could you explain the original problem resulting in these changes?

Reply:
When the browser was wide enough for the non-mobile menu, clicking a nav item once would result in the "opened" state being toggled. Meaning that it was then "opened". Switching to the mobile view (e.g. by making your browser window smaller or turning your phone) would then have the menu half open when it should be closed. The actual nav items should only ever close the menu, never open it, so this is semantically more correct even.